vector/src/main/java/im/vector/app/features/form/FormEditTextItem.kt

```diff
@@ -64,20 +68,38 @@ abstract class FormEditTextItem : VectorEpoxyModel<FormEditTextItem.Holder>() {
     @EpoxyAttribute(EpoxyAttribute.Option.DoNotHash)
     var editorActionListener: TextView.OnEditorActionListener? = null
 
+    @EpoxyAttribute(EpoxyAttribute.Option.DoNotHash)
+    var onFocusChange: ((Boolean) -> Unit)? = null
+
+    @EpoxyAttribute
+    var inputPrefix: String? = null
+
+    @EpoxyAttribute
+    var inputSuffix: String? = null
```

Comment:
Why not using the names than the TextInputLayout names? `prefixText` and `suffixText` ?

Reply:
done